app/views/contexts/summary/group.py

```diff
@@ -116,16 +116,15 @@ def _build_blocks_and_links(
                             list_collector_block.list_summary_element(summary_item)
                         )
                         blocks.extend([list_summary_element])
-                        links["add_link"] = SummaryLink(
+                        self.links["add_link"] = SummaryLink(
```

Comment:
Any need for `SummaryLink` instead of updating the `Link` class?.

Reply:
Not if it is defaulted?

Reply:
Even when I default like:
```
@dataclass
class Link:
    text: LazyString
    url: str
    target: Optional[str] = "_blank"
    attributes: Optional[dict] = field(default_factory=dict)
```
I get multiple errors where `'attributes': {}` is expected in footer links. So I went with another dataclass for summary link.

Reply:
Ok, done it for links using `as_dict()` method in `Link` class.